provisioning/templates-api/azure.sh

```diff
@@ -22,6 +22,7 @@ CLUSTER_NAME=$(az deployment group show \
 az aks get-credentials --name "$CLUSTER_NAME" --resource-group "$RESOURCE_GROUP" --overwrite-existing
 
 # Common part of the deploy
+export ETCD_STORAGE_CLASS_NAME=
```

Comment:
Nebolo by tu lepsie dat nejaku konkretnu class, aby sa to nerozbilo ked sa zmeni default? Neviem co tam teraz je.

Reply:
Ked je pre AWS konkretna hodnota, tak by som ju dal aj pre Azure.

Reply:
Mrknu co je za default v Azure

Reply:
je tam `default`
<img width="1376" alt="Snímek obrazovky 2023-01-08 v 17 46 08" src="https://user-images.githubusercontent.com/1696274/211225036-c354da07-9ed2-489b-ad64-1abe126d4411.png">

Reply:
ale jak už jsme psal výše, kvůli strategy to nejde aktualizovat ani když jde o stejnou class
```bash
2023-01-09T00:06:32.2968875Z Error: UPGRADE FAILED: cannot patch "templates-api-etcd" with kind StatefulSet: StatefulSet.apps "templates-api-etcd" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden
```

https://dev.azure.com/keboola-prod/Keboola%20Connection%20Azure/_releaseProgress?_a=release-environment-logs&releaseId=30706&environmentId=87210

Reply:
bez uvedení class to projde a nic se nezmění, takže bych to neřešil https://dev.azure.com/keboola-prod/Keboola%20Connection%20Azure/_releaseProgress?_a=release-pipeline-progress&releaseId=30707

Reply:
pro local platí to samé https://github.com/keboola/keboola-as-code/actions/runs/3869558996/jobs/6595729759#step:13:833

Reply:
Pro PV `data-buffer-etcd-1` vidím [Resource ID](https://portal.azure.com/#@keboolaconnection.onmicrosoft.com/resource/subscriptions/96fd974a-f038-4167-9966-0fd17d33e82d/resourceGroups/mc_kbc-testing-azure-east-us-2_kbc-aks-nwbdw35shzpvm_eastus2/providers/Microsoft.Compute/disks/pvc-1b6810e1-cfbf-45dc-9d5c-844f178db156/overview)
```bash
/subscriptions/96fd974a-f038-4167-9966-0fd17d33e82d/resourceGroups/mc_kbc-testing-azure-east-us-2_kbc-aks-nwbdw35shzpvm_eastus2/providers/Microsoft.Compute/disks/pvc-1b6810e1-cfbf-45dc-9d5c-844f178db156
```

To je class `Standard SSD LRS`, což taky není žádná sláva dle aktuálního nastavení.

<img width="1412" alt="Snímek obrazovky 2023-01-08 v 18 46 17" src="https://user-images.githubusercontent.com/1696274/211227149-0f98be47-db58-4d7b-948d-1f15d4f11ca8.png">